scripts/resource/basic_discovery.rc

```diff
@@ -80,6 +80,10 @@ print_line("starting discovery scanners ... stage 1")
 print_line("============================================")
 print_line("")
 
+# Temp variable to store space separated RHOSTS
+nmap_rhosts = framework.datastore['RHOSTS'].gsub(',',' ')
+run_single("set RHOSTS #{nmap_rhosts}")
```

Comment:
Is this line needed? 👀

Reply:
yes, without it the MSF validate RHOSTS error would come up, even the auxiliary scan requires space separated values not comma separated so...

Reply:
Ah, thanks. I wonder if it gets treated as a URL when there are no spaces and CIDR is defined. Will try and take a quick look tomorrow

Reply:
@adfoster-r7 Any updates on the test cases? I tested the when CIDR is defined but couldn't seem to find anything out of the ordinary.